src/NovelRT/CMakeLists.txt

```diff
@@ -161,11 +161,38 @@ if(WIN32)
       Winmm
   )
 
+  set(CMAKE_INSTALL_PREFIX "C:/Program Files/NovelRT" CACHE PATH "..." FORCE)
```

Comment:
No.

Reply:
Please provide an alternative @FiniteReality. "No." doesn't help anything other than you don't like it.

Reply:
Setting `CMAKE_*` variables is a *massively* bad idea. It should be set by the CI infrastructure or the user, and *never ever* overridden.

Reply:
I believe the problem is that its defaulting to Program Files (x86). We can't have that. This is a 64 bit software.

Reply:
Then tell users to specify `-DCMAKE_INSTALL_PREFIX=path/to/a/better/location`? Programs cannot install there anyway without admin permissions. Our build script should not assume you're running as an admin.

Reply:
This is the feedback I was looking for as I wasn't sure we'd ever really want to install in the Program Files folders like CMake wants to do by default...

@RubyNova shall we just make sure to specify this in the readme?

Reply:
I suppose so? I have no idea.

Reply:
Wait what? Debugging on windows is broken?

Reply:
> Wait what? Debugging on windows is broken?

Was this a misplaced question?